ui/src/testing/stubs/runtimes-api-stub.ts

```diff
@@ -1,12 +1,8 @@
 import { RuntimesApi } from 'notebooks-generated/fetch';
 
-import { stubNotImplementedError } from './stub-utils';
-
 export class RuntimesApiStub extends RuntimesApi {
   constructor() {
-    super(undefined, undefined, (..._: any[]) => {
-      throw stubNotImplementedError;
-    });
+    super(undefined);
```

Comment:
This ultimately extends BaseApi which only takes one parameter: a configuration object.